src/app/kuma/locales/en-us/index.yaml

```diff
@@ -94,10 +94,13 @@ http:
       tls: TLS
       mtls: mTLS
       mTLS: mTLS
+      id: ID
       globalInstanceId: Global instance ID
       controlPlaneInstanceId: CP instance ID
+      zoneInstanceId: Zone Leader Instance ID
       connectTime: Connected
       disconnectTime: Disconnected
+      version: Version
```

Comment:
These are now part of the `http.api.property` i18n string. Means 2 things:

1. We don't have to add them in other modules as these are commonly used across the entire app.
2. The terms are consistent through out the entire app (see point 1)

This is also related to https://github.com/kumahq/kuma-gui/issues/2814 and https://github.com/kumahq/kuma-gui/issues/2431.

You'll notice I possibly seem overly cautious about adding i18n at the moment even though we really want our entire app to be 100% i18n'ed. I'd just really like to find/figure a good pattern/tooling so we don't just keep digging the current hole deeper. The hole is currently on the wrong place, and once we find the right place, we are gonna have to fill this one back in again. Possibly bad analogy! 😆

Reply:
I like the idea to have the entire app i18n'ed 👍 but you are right that we have to find a good pattern and structure 🙂 what do you mean with "the hole is currently on the wrong place"?

Reply:
A while back we put i18n in place (we bought shovels), we just went crazy digging everywhere instead of finding the right place to dig first. Once we've found the right place, we can dig there, but we'll have to also fill up all the places where we dug in the wrong place i.e. the refactor. Not sure if its a good idea to continue with the analogy! 😆 

---

To give some history: I was really keen to get i18n in place, but at first **purely so we could do "machine to human" translations**. So for example we use it a lot for our tabs where we use the `route.name` as the i18n key for the text in the tab, it translates the "machine" `route.name` to "human" text (not necessarily of any particular language). A lot of this "machine" to "human" was converting HTTP API properties/values to "human" (e.g. `mtls` to `mTLS`). I think we did this right by having `http.api.property`, `http.api.value` keys to have centrally reference-able human texts for these things.

Thing is, once we added i18n, we then went ahead and did lots of localisation on things that weren't to do with "machine" to "human" translation, and we did the i18n strings using completely different conventions, so there is no consistency anywhere. In some places we didn't think about using the `http.api.property` section of our i18n strings. i.e. we dug lots of different holes all over the place, with very few in the right place. i.e. the convention (or lack of convention) we have is all wrong. We are going to have to go back and change it all, the more there is the more we'll have to change.

So basically exactly what you said "we have to find a good pattern and structure", a lot of it doesn't have that as yet, so once we find that pattern (the sooner the better) we'll have to go through and apply that pattern to all of the places where we don't use it.

Hope that helps! Analogies are sometimes a bad idea 😬 😆

Reply:
Alright, got it, thank you 🙂 👍